src/components/BulkEditList/BulkEditList.js

```diff
@@ -149,8 +149,10 @@ export const BulkEditList = () => {
           values={{ fileName: confirmedFileName || initialFileName }}
         />
       );
+    } else if (isQueryTabWithPreview) {
+      return <FormattedMessage id="ui-bulk-edit.meta.query.title" />;
     } else return <FormattedMessage id="ui-bulk-edit.meta.title" />;
-  }, [confirmedFileName, initialFileName, isIdentifierTabWithPreview, history.location.search]);
+  }, [confirmedFileName, initialFileName, isIdentifierTabWithPreview, isQueryTabWithPreview, history.location.search]);
```

Comment:
Could you fix eslint error please?

Reply:
Done! @artem-blazhko please take a look again